client/sites/staging-site/components/staging-site-card/card-content/style.scss

```diff
@@ -0,0 +1,5 @@
+.staging-site-card__loading-bar {
+	width: 100% !important;
```

Comment:
I wanted to avoid using `!important` here, but haven't found a cleaner approach.